### PR TITLE
Fix tooltip arrow style and position

### PR DESCRIPTION
**Pull request by phalkunz**
Fixes https://github.com/silverstripe/silverstripe-admin/issues/229

### CODE REVIEW

**Comment by dhensby** [comment]
there are some linting issues with this (and it needs rebase/rebuild)

**Comment by flamerohr** [comment]
Arrow is on the wrong side when in mobile
screen shot 2017-10-02 at 11 39 39 am

File: client/src/components/Tooltip/Tooltip.scss [hunk 1]
```diff
@@ -11,3 +11,77 @@
   }
 }
 
+/**
+ * Tooltip arrow style below is copied (unless stated otherwise) from Bootstrap
+ * v4 beta because there's a missmatch between classname in react-bootstrap and
+ * its counterpart in Bootstrap v4.
```
Comment (flamerohr) [inline client/src/components/Tooltip/Tooltip.scss]: could this use `@extend` so it's not hardcoded-copying? use `// sass-lint:disable-block extend-rule-name` for linting with the comment above

Comment (phalkunz) [reply client/src/components/Tooltip/Tooltip.scss]: This requires us to extends `.tooltip .arrow` but unfortunately we can't extend a nested selector.

**Pull request merged**